### DISCUSSION TITLE
Codespace. Workbench failed to connect

### DISCUSSION THREAD

**Discussion by KennethSmithRSA** [community/community#43303] [Codespaces]
### Select Topic Area

Bug

### Body

![VSCODE-codespace-error](https://user-images.githubusercontent.com/64465363/210563220-eeed0ede-caad-451b-8934-5a7b20a26afa.PNG)
Hey guys, i was using vs code space this morning and it was working fine, however, when i can back from lunch this error would constantly appear when i try to access my code space. I am using brave browser but i does the same thing on chrome.

**Comment by Ankush11903** [comment]
TRY RESTATING YOUR PC.
it might work.

**Comment by mishra-prajjawal** [comment]
Hello @CrimsonR3ap3r  👋 to address your problem I believe that your subscription may have been expired. 
If it is not then try clearing browser cache and delete all cookies. 
Now login and try to connect. 
Other wise you can try connecting through GitHub codespaces extension on VSCode

**Comment by erikaellr** [comment]
Hi @CrimsonR3ap3r, we're thrilled you're using Codespaces, but sorry to hear you were running into this problem. Does the problem happen with just a single codespace, or does it also repro when you create and connect to a brand new codespace? If you can successfully connect to a newly-created codespace, you could [export your changes to a branch](https://docs.github.com/en/codespaces/troubleshooting/exporting-changes-to-a-branch) from the codespace that is giving you the error to be able to safely delete the problem codespace and continue your work on that branch on a new codespace. If that doesn't resolve the problem for you, or you continue to run into this error with multiple codespaces, you can open a ticket on our [Support page](https://support.github.com/contact).
